code/modules/mob/living/carbon/human/species/station/station.dm

```diff
@@ -209,16 +205,18 @@
 	flesh_color = "#AFA59E"
 	base_color = "#333333"
 
-	//reagent_tag = IS_TAJARA //VOREStation Removal
+	reagent_tag = IS_TAJARA
```

Comment:
Unless you understand what this does, change this back, because I'm pretty sure all this does is make Tajaran sick for eating non-meats, which is aggravating as hell.

Reply:
Effects are:
- Makes Coffee slightly toxic to Tajara (commented out elsewhere, though)
- Makes alcohol's effects stronger to Tajara
- Hyperzine and... something else (toxin/stimm) are removed faster from Tajaran blood